tests/python/unittest/test_operator.py

```diff
@@ -5919,6 +5919,31 @@ def f(x, a, b, c):
         check_numeric_gradient(quad_sym, [data_np], atol=0.001)
 
 
+@with_seed()
+def test_histogram():
+    def f(x, bins=10, range=None):
+        return np.histogram(x, bins, range=range)
+
+    shape = (3, 3, 3)
```

Comment:
Randomize shape and test ndarrays from 1D-5D.

Reply:
Done